DDG4/python/DDSim/Helper/Meta.py

```diff
@@ -61,6 +66,38 @@ def parseEventParameters(self):
         intParameters[pname] = pvalue
     return stringParameters, intParameters, floatParameters
 
+  def parseRunParameters(self):
```

Comment:
I have the feeling this could be merged with parseEventParameters?
```
def parseMetaParameters(self, parType=event):
   # ...
   for p in getattr(self, parType + "Parameters"):
...
?